main.go

```diff
@@ -142,6 +145,10 @@ func main() {
 
 	log.Info("started")
 
+	if *afFocusNow > 0 {
+		lvs.AFFocusAfter(time.Duration(*afFocusNow) * time.Second)
```

Comment:
Related change to the AFFocusAfter().

```suggestion
		go lvs.AFFocusAfter(time.Duration(*afFocusNow) * time.Second)
```

Reply:
done